pinot-query-runtime/src/test/java/org/apache/pinot/query/QueryServerEnclosure.java

```diff
@@ -112,7 +112,6 @@ public int getPort() {
   public void start()
       throws Exception {
     PinotConfiguration configuration = new PinotConfiguration(_runnerConfig);
-    _queryRunner = new QueryRunner();
```

Comment:
good catch

Reply:
good catch, we can also make _queryRunner final